### PR TITLE
Add Network by Filesize

### PR DESCRIPTION
**Pull request by purukaushik**
- Added rounding to 10th place decimal
- Added KB-grain fileSizes
- fixed front-end for endpoint /networksbyfilesize

### CODE REVIEW

File: routes/index.js [hunk 1]
```diff
@@ -157,7 +157,7 @@ router.get('/networksbyfilesize', function(req,res) {
                     {
                         $group: {
                             _id : "$adNetwork",
-                            fileSize : { $avg : "$fileSize"}
+                            fileSize : {$avg : "$fileSize"},
```
Comment (FrancescoSTL) [inline routes/index.js]: was the adding of this comma intentional?

**Comment by FrancescoSTL** [comment]
It would be good to make the titles a bit more descriptive in the future.

Outside of that, looks like I'm still getting 0KB when loading locally
screen shot 2016-08-04 at 11 10 48 pm

**Comment by FrancescoSTL** [comment]
^ it could be that we've still got older versions in the wild which are posting strings to our fileSize field, but I'm unsure. Is this PR complete?

**Comment by purukaushik** [comment]
I'll cleanup local db once and retest.

**Pull request merged**